app/src/main/scala/org/alephium/explorer/persistence/queries/TransactionQueries.scala

```diff
@@ -92,10 +93,10 @@ object TransactionQueries extends StrictLogging {
   def updateTransactionPerAddressAction(outputs: Seq[OutputEntity], inputs: Seq[InputEntity])(
       implicit ec: ExecutionContext): DBActionRW[Seq[InputEntity]] = {
     for {
-      _              <- insertTxPerAddressFromOutputs(outputs)
       inputsToUpdate <- insertTxPerAddressFromInputs(inputs, outputs)
     } yield inputsToUpdate
```

Comment:
After the deletion this function is calling another function with the same parameters. Should this function be removed?

Reply:
everything will be removed in https://github.com/alephium/explorer-backend/pull/282 :)